wca-regulations.md

```diff
@@ -69,7 +69,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 2b) Competitors below the age of 18 must obtain consent from their parent(s)/guardian(s) to register and compete.
 - 2c) Competitors register by providing all information required by the organization team (including: name, country, date of birth, gender, contact information, selected events).
     - 2c1) A competitor is not eligible to compete without a completed registration, as determined by the organization team.
-    - 2c2) Returning competitors must provide personal information consistent with their WCA profile. If a competitor wishes to change or update their personal information between competitions (e.g. a change of nationality, name, or gender), they must [contact the WCA Results Team](https://www.worldcubeassociation.org/contact/website).
+    - 2c2) Returning competitors must provide personal information consistent with their WCA profile. If a competitor wishes to change or update their personal information between competitions (e.g. a change of nationality, name, or gender), they must [contact the WCA Results Team](https://www.worldcubeassociation.org/contact/website) either directly or through their local WCA Delegate.
```

Comment:
```suggestion
    - 2c2) Returning competitors must provide personal information consistent with their WCA profile. If a competitor wishes to change or update their personal information between competitions (e.g. a change of nationality, name, or gender), they must [contact the WCA Results Team](https://www.worldcubeassociation.org/contact/website) either directly or through a WCA Delegate.
```

Competitors may have reasons to contact a non-local Delegate, for example when that Delegate is delegating a competition they attend, or when they wish to contact a trusted Delegate for a sensitive change. As the regulation is a  must it should not be overly restrictive.